types/selenium-webdriver/bidi/evaluateResult.d.ts

```diff
@@ -0,0 +1,39 @@
+interface IEvaluateResultType {
+    SUCCESS: "success";
+    EXCEPTION: "exception";
+}
+
+const EvaluateResultType: IEvaluateResultType;
+
+declare class EvaluateResultSuccess<T> {
+    constructor(realmId: string, value: T);
+    resultType: "success";
+    realmId: string;
+    result: T;
+}
+
+declare type ExceptionShape = {
```

Comment:
Just curious, why is ExceptionShape needed? I don't have typescript knowledge, apologies due to knowledge gap.

Reply:
@pujagani  - I wanted to type the intermediate for the constructor argument to exception details, and there's some little differences (I think). For example, this uses the 'in' operator, which indicates that there is the possibility for an 'undefined' value on the constructor keys. Conversely, this assigns to null on the class as a fallback, so it will never have 'undefined' in the key union type. As far as that naming, I have a poor naming problem 😅 This should probably be named ExceptionDetailsInput or ExceptionDetailsConstructorOpts.

A lot of what I'm doing here is rooted in ignorance and old bad habits